egs/TAC/local/tac_dataset.py

```diff
@@ -0,0 +1,145 @@
+from torch.utils.data import Dataset
```

Comment:
I wonder why not adding this is asteroid/data? 
Is there a reason? Do you feel it's too recipe specific?

Reply:
This dataset is problematic if you remember due to lack of spatial diversity in mics and the limited noise types.
I would not add it to the toolkit IMO.

Reply:
Ok, cool